### DISCUSSION TITLE
Bun v1.0.23

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#8200] [Announcements]
To install Bun v1.0.23:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.23:
```bash
bun upgrade
```

**[Read Bun v1.0.23's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.23)**

## Thanks to 12 contributors who made this release possible!

- [@aayushbtw](https://github.com/aayushbtw)
- [@cirospaciari](https://github.com/cirospaciari)
- [@dylan-conway](https://github.com/dylan-conway)
- [@Electroid](https://github.com/Electroid)
- [@eriklangille](https://github.com/eriklangille)
- [@gvilums](https://github.com/gvilums)
- [@hugo-syn](https://github.com/hugo-syn)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@kaioduarte](https://github.com/kaioduarte)
- [@LukasKastern](https://github.com/LukasKastern)
- [@nektro](https://github.com/nektro)
- [@paperdave](https://github.com/paperdave)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.23'>Bun v1.0.23</a>.</em>

**Comment by TomasHubelbauer** [comment]
For anyone interested in the same - currently the DB import doesn't do `create: true` by default nor can it accept `create: true` in the `with` import assertion. I asked for the support for the latter here: https://github.com/oven-sh/bun/issues/8202

**Comment by Jonatthu** [comment]
This is breaking vscode debug task with
![telegram-cloud-photo-size-1-4985961181432885682-y](https://github.com/oven-sh/bun/assets/3742176/8c8eed1b-a609-4af3-a3d2-41f71c730eec)
It works again when I downgrade to 1.0.22
